checker/default_checks.go

```diff
@@ -34,7 +34,6 @@ var optionalChecks = map[string]BackwardCompatibilityCheck{
 	"response-property-enum-value-removed":  ResponseParameterEnumValueRemovedCheck,
 	"response-mediatype-enum-value-removed": ResponseMediaTypeEnumValueRemovedCheck,
 	"request-body-enum-value-removed":       RequestBodyEnumValueRemovedCheck,
-	"api-path-added":                        APIAddedCheck,
```

Comment:
I prefer to have all info-level checks in one place (defaultChecks) for consistency, rather than having some of them in optionalChecks.